lib/schlepp/format/base.rb

```diff
@@ -70,9 +69,17 @@ def process!
       # Glob support to pull file list for processing.
       def retrieve_file_list
         if name.is_a? Array
-          name.flat_map { |n| Dir.glob(n) }
+          if Format.cwd == ''
+            name.flat_map { |n| Dir.glob(n) }
```

Comment:
no need for flat map here - it's only useful if what you're getting is an array of arrays and you want to flatten it down a level.